links/sm.json

```diff
@@ -0,0 +1,63 @@
+[
+{
+	"Name":        "Handlingar",

```

Comment:
please make all keys lowercase :) (go's json parsing is case insinsetive (which is a little cursed))